.github/CONTRIBUTING.md

```diff
@@ -16,6 +16,17 @@ Include a clear example of the current behaviour with (as much as possible):
 - Screenshots of the behaviour
 - Information on any triage you've attempted
 
+### In case of a breaking issue in Production
+
+Our first priority is to guarantee a stable version in Production (deployed from `main` in our case) at all times, but issues can always come about and in case they do, we have a Rollback Plan!
```

Comment:
but issues can always come about and in case they do

this bit sounds a bit funny maybe rephrase

Reply:
Edited in the latest commit